index.html

```diff
@@ -1,414 +1,482 @@
-				</p>
-			</section>
+                <p>in Turtle, and</p>
 
-			<section>
-				<h2>Serializations</h2>
+                <pre class=nohighlight>
+						"ex:example" : {
+						    "@value" : "<bdo dir="ltr">&#x05E4;&#x05E2;&#x05D9;&#x05DC;&#x05D5;&#x05EA; &#x05D4;&#x05D1;&#x05D9;&#x05E0;&#x05D0;&#x05D5;&#x05DD;, W3C</bdo>",
+						    "@language" : "he-x-d-rtl"
+						}
+					</pre>
 
-				<p>
-					In terms of serializations, this change would be completely transparent for RDF. Ie, the example used before is:
-				</p>
+                <p>in JSON-LD.</p>
+            </section>
+
+            <section>
+                <h2>Pros and Cons</h2>
+
+                <p>
+                    A major advantage of this approach are similar to <a href=#extending-bcp></a>: it is fully transparent to current RDF deployments, and it affects only application layers that do display the literal values for humans and it also works for parsers “out of the box. Because it is explicitly restricted to some specifications, it can (and should) be ignored by HTML, CSS, or other usage areas of [[bcp47]].
+                </p>
+
+                <p>
+                    Just as for <a href=#extending-bcp></a>, a major disadvantage is that [[bcp47]] is already a very complex specification, in terms of the rich "metadata" that it assigns to languages. It is also not clear whether it "fits" the purpose of&nbsp;[[bcp47]], which is to describe metadata about language, scripts used, etc., and not a feature that is relevant for the <em>display</em> of that string. Furthermore, although it is restricted to a specific family of specification, its usage may “spill over” to other areas, creating confusion.
+                </p>
+                <p>
+                    Finally, just as for <a href=#extending-bcp></a>, the interplay between current data deployment and the assignment of such extended language tags may not be obvious.
+                </p>
+            </section>
+        </section>
+        <section id=script-subtag>
+            <h2>Rely on current BCP47 only</h2>
+
+            <p>
+				A different approach stems from the observation that the current BCP-47 can be used, unchanged, to express the necessary base direction using, if necessary, the “script” subtag. For example, the tag <code>he-Hebr</code> reinforces, through the presence of the <code>Hebr</code> subtag, that the language is not only Hebrew, but the string uses the Hebrew script, i.e., it <em>is</em> right-to-left. In other words:
+			</p>
 
-				<pre class=nohighlight>
-					[] ex:example "<bdo dir="ltr">&#x05E4;&#x05E2;&#x05D9;&#x05DC;&#x05D5;&#x05EA; &#x05D4;&#x05D1;&#x05D9;&#x05E0;&#x05D0;&#x05D5;&#x05DD;, W3C</bdo>@he-d-rtl .
+            <pre class=nohighlight>
+					[] ex:example "<bdo dir="ltr">&#x05E4;&#x05E2;&#x05D9;&#x05DC;&#x05D5;&#x05EA; &#x05D4;&#x05D1;&#x05D9;&#x05E0;&#x05D0;&#x05D5;&#x05DD;, W3C</bdo>"@he-Hebr .
```

Comment:
Again, he-Hebr -> he, or use Azeri.

Reply:
Can you give me an azeri example?

Reply:
that's the `az-Arab` example in the earlier comment.  See https://r12a.github.io/app-subtags/?lookup=az-Arab